src/main/java/org/jacksonlaboratory/controller/SearchController.java

```diff
@@ -31,13 +31,13 @@ public SearchController(TermService termService) {
 	public SearchDto search(
 			@QueryValue("q") @Schema(minLength = 3, maxLength = 250, type = "string", pattern = ".*")  String query,
 			@QueryValue(value = "page", defaultValue = "0") @Schema(maxLength = 1000, type = "number") int page,
```

Comment:
I like 0-based pages, it makes the code elegant `stream().skip(page).limit(limit)` which I do a lot.